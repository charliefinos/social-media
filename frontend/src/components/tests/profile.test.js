import React from 'react'
import ReactDOM from 'react-dom'
import Basic from '../Basic'

import Enzyme, { shallow, render, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17'

Enzyme.configure({ adapter: new Adapter() })

it('reders correctly enzyme', () => {
    const wrapper = shallow(<Basic />)

    expect(toJson(wrapper)).toMatchSnapshot();
})