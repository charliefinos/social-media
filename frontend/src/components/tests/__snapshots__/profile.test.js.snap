// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reders correctly enzyme 1`] = `
<div>
  <h1>
     Basic Test
  </h1>
</div>
`;
